test/plcrVotingChallenge/claimVoterReward.js

```diff
@@ -58,20 +59,6 @@ contract('Registry', (accounts) => {
       );
     });
 
-    it('should revert if challenge does not exist', async () => {
```

Comment:
What's the reasoning for removing this one?

Reply:
the function for claiming voter reward was moved to the challenge contract, so now there's no way to call it unless the challenge exists